modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Settings.tsx

```diff
@@ -0,0 +1,347 @@
+								</span>
+							</ClayTooltipProvider>
+						</div>
+
+						<div>
+							{Liferay.Language.get(
+								'default-visualization-mode-explanation'
+							)}
+						</div>
+					</ClayLayout.Col>
+
+					<ClayLayout.Col size={3}>
+						<Picker
+							aria-labelledby="view-mode"
+							id="view-mode-picker"
+							items={visualizationModes}
+							onSelectionChange={(option: React.Key) => {
+								if (
+									option ===
+									NOT_CONFIGURED_VISUALIZATION_MODE.name
+								) {
+									onActiveSectionChage(1);
+								}
+								else {
+									setDefaultView(option as string);
+								}
+							}}
+							placeholder={NOT_CONFIGURED_VISUALIZATION_MODE.name}
+							selectedKey={defaultView}
+						>
+							{visualizationModes.length ? (
+								({label, name, thumbnail}) => (
+									<Option key={name} textValue={label}>
+										<ClayIcon
+											className="mr-3"
+											symbol={thumbnail}
+										/>
+
+										{label}
+									</Option>
+								)
+							) : (
+								<DropDown.Group
+									header={Liferay.Language.get(
+										'not-configured'
+									)}
+								>
+									<Option
+										key={
+											NOT_CONFIGURED_VISUALIZATION_MODE.name
+										}
+										textValue={
+											NOT_CONFIGURED_VISUALIZATION_MODE.name
+										}
+									>
+										<ClayLayout.Row>
+											<ClayLayout.Col className="align-self-center">
+												<ClayIcon
+													symbol={
+														NOT_CONFIGURED_VISUALIZATION_MODE.thumbnail
+													}
+												/>
+											</ClayLayout.Col>
+
+											<ClayLayout.Col size={10}>
+												<Text size={3}>
+													{
+														NOT_CONFIGURED_VISUALIZATION_MODE.label
+													}
+												</Text>
+											</ClayLayout.Col>
+										</ClayLayout.Row>
+									</Option>
+								</DropDown.Group>
+							)}
+						</Picker>
+					</ClayLayout.Col>
+				</ClayLayout.Row>
+			</ClayLayout.SheetSection>
+
+			<ClayLayout.SheetSection className="invisible mt-4">
+				<h3 className="sheet-subtitle">
+					{Liferay.Language.get('user-customization')}
+				</h3>
+
+				<ClayLayout.Row className="align-items-center justify-content-between">
+					<ClayLayout.Col size={10}>
+						<div>
+							<label
+								htmlFor="custom-view-toggle"
+								id="custom-views"
+							>
+								{Liferay.Language.get('enable-custom-views')}
+							</label>
+						</div>
+
+						<div>
+							{Liferay.Language.get(
+								'enable-custom-views-explanation'
+							)}
+						</div>
+					</ClayLayout.Col>
+
+					<ClayLayout.Col
+						className="d-flex justify-content-end"
+						size={2}
+					>
+						<ClayToggle
+							aria-labelledby="custom-view-toggle"
+							id="custom-view-toggle"
+							onKeyDown={(_event) => {
+								setEnableCustomView(!enableCustomView);
+							}}
+							onToggle={setEnableCustomView}
+							toggled={enableCustomView}
+						/>
+					</ClayLayout.Col>
+				</ClayLayout.Row>
```

Comment:
As I was tinkering with the UI I've created everything in the page and made it invisible https://github.com/liferay-frontend/liferay-portal/pull/4063/files/f8812befc9170226bc474ca83047a723fd211ad3#diff-57c4bb2ff99069fe9196d1fa92f187e2efbe04603029158f0bc1cca2228d5a44R289.

Anyways, it could be removed without problem.